ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward.hpp

```diff
@@ -122,19 +66,16 @@ struct ExecuteTernaryBackwardLerp {
 }  // operations::ternary_backward
 
 //type 1
-constexpr auto addcmul_bw = ttnn::register_operation_with_auto_launch_op<
+constexpr auto addcmul_bw = ttnn::register_operation<
```

Comment:
Can you show me where the test is for this operation? I'd just like to make sure that removing the auto launch is indeed the correct thing to do here.

Reply:
@patrickroberts the test is located at `tests/ttnn/unit_tests/operations/backward/test_backward_addcmul.py`
I have verified the tests and they are passing